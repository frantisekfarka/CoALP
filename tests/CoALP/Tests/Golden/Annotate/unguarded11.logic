q(X,X,X1,X2):-p(X1,X2,X1,X2).
p(X,X,X1,X2):-q(X1,X2,X1,X2).
