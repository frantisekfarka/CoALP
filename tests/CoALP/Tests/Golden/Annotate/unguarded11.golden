q(V_2i, V_2i, V_3i, V_4i, transform-func-1(V_8c)) :- p(V_3i, V_4i, V_3i, V_4i, V_8c).
p(V_5i, V_5i, V_6i, V_7i, transform-func-2(V_9c)) :- q(V_6i, V_7i, V_6i, V_7i, V_9c).
